src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -232,6 +260,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String shadowEmail = userService.getCurrentUser().getEmail();
     String shadowId = getUserId();
+
```

Comment:
What's with all these blank lines?

Reply:
used them to see code clearer, just removed